templates/resources.md.tmpl

```diff
@@ -0,0 +1,26 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: "{{ index .Metadata "service"}}"
```

Comment:
Adds generic resource/data-source templates to expect and pull in metadata. The existing resource level custom templates will need to be updated as well.